WordPress/src/main/java/org/wordpress/android/ui/pages/PageItem.kt

```diff
@@ -12,43 +12,58 @@ import org.wordpress.android.ui.pages.PageItem.Action.VIEW_PAGE
 import org.wordpress.android.ui.pages.PageItem.Type.DIVIDER
 import org.wordpress.android.ui.pages.PageItem.Type.EMPTY
 import org.wordpress.android.ui.pages.PageItem.Type.PAGE
+import java.util.Date
 
 sealed class PageItem(open val type: Type) {
     abstract class Page(
         open val id: Long,
         open val title: String,
+        open val date: Date,
         open val labels: List<Int>,
         open var indent: Int,
+        open var imageUrl: String?,
         open val actions: Set<Action>,
         open var actionsEnabled: Boolean
     ) : PageItem(PAGE)
 
     data class PublishedPage(
         override val id: Long,
         override val title: String,
+        override val date: Date,
         override val labels: List<Int> = emptyList(),
         override var indent: Int = 0,
+        override var imageUrl: String? = null,
         override var actionsEnabled: Boolean = true
-    ) : Page(id, title, labels, indent, setOf(VIEW_PAGE, SET_PARENT, MOVE_TO_DRAFT, MOVE_TO_TRASH), actionsEnabled)
+    ) : Page(id, title, date, labels, indent, imageUrl, setOf(VIEW_PAGE, SET_PARENT, MOVE_TO_DRAFT, MOVE_TO_TRASH),
```

Comment:
I know this is not really related to this PR, but how would you feel about adding the name arguments to the constructors in this file? Since there are a lot of arguments, I think it'd increase the readability of them as well as making them less error prone since the order won't matter.

_P.S:_ Just in case you didn't know about it, AS has a helper for this. Just select the `(` of the constructor, do an `Alt + Enter` and select `Add names to call arguments`.

![screen shot 2019-01-16 at 5 12 35 pm](https://user-images.githubusercontent.com/662023/51282035-1b09a200-19b2-11e9-8a28-045c293aff6b.png)

Reply:
Good idea, thanks for the suggestion 👍.